reference/tools/env/virtualrunenv.rst

```diff
@@ -13,6 +13,11 @@ the compiled executables and applications. The information is obtained from:
     - Also automatically deduced from the ``self.cpp_info`` definition of the package, to define ``PATH``,
       ``LD_LIBRARY_PATH``, ``DYLD_LIBRARY_PATH`` and ``DYLD_FRAMEWORK_PATH`` environment variables.
 
+.. note::
```

Comment:
Thanks for the contribution.
I think the clarification is good, but probably doesn't need to be highlighted as a note, as the vast majority of users won't have this problem. Maybe as an extra bullet in the list above?

Reply:
Thanks for the suggestion, I split the above bullet point to make it clearer but stay more in line with the "The information is obtained from:".